os.cpp

```diff
@@ -1,380 +0,0 @@
-/*
- * os.cpp
- *
- * Baker, Ballard, Jager-Kujawa
- * CSC 341
- * Spring 2016
- */
-
-#include "os.hpp"
-
-queue<User> readyQueue = queue<User>();
-queue<User> blockedQueue = queue<User>();
-User currentUser;
-bool semaphore = unlocked;
```

Comment:
This can no longer simply be a boolean; it must be a boolean array of size 64, each value referencing a frame of memory.